src/Microsoft.Extensions.Primitives/ChangeTokenHelper.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.Primitives
+{
+    /// <summary>
+    /// Propagates notifications that a change has occured.
+    /// </summary>
+    public static class ChangeTokenHelper
```

Comment:
At first glance it seems that the best name for this class would be just ChangeToken, but not sure if it would make sense to add a static class with that name or a base implementation for the interface with these as extra static methods.